tests/parametric/test_dynamic_configuration.py

```diff
@@ -495,25 +516,22 @@ def test_tracing_client_tracing_tags(self, library_env, test_agent, test_library
 
     @parametrize("library_env", [{**DEFAULT_ENVVARS}])
     def test_capability_tracing_sample_rate(self, library_env, test_agent, test_library):
-        """Ensure the RC request contains the trace sampling rate capability.
-        """
+        """Ensure the RC request contains the trace sampling rate capability."""
         test_agent.wait_for_rc_capabilities([Capabilities.APM_TRACING_SAMPLE_RATE])
 
-    @irrelevant(library="golang", reason="The Go tracer doesn't support automatic logs injection")
+    @irrelevant(context.library in ("cpp", "golang"), reason="Tracer doesn't support automatic logs injection")
```

Comment:
Is it something that will be supported one day? if yes, the good decorator is `missing_feature`. `@irrelevant` must be used only if the feature will never be implemented.

Reply:
I do not think the feature will be implemented, `@irrelevant` is the right decorator here.